session4/src/com/itschool/homework/Challenge6.java

```diff
@@ -0,0 +1,26 @@
+package com.itschool.homework;
+
+public class Challenge6 {
+
+    public static void main(String[] args) {
+
+        int d = 5;
+        int e = 4;
+
+        System.out.println(d>e);
+        System.out.println(d<e);
+        System.out.println(d==e);
+
+        if (d >= 0 && d <= 100) {
+            System.out.println("d is in the range");
+        }
+
+        if (e >= -10 && e <= 3) {
+            System.out.println("This is not supposed to be displayed");
```

Comment:
and how can I be informed that `e` is not in range?
Also, it would be nice that these numbers are entered by the user from keyboard and not hard-coded in the program.

Reply:
I do not know how to improve my code here :( Please, help me.

Reply:
sure! In order to enter these numbers from keyboard you can apply the same logic used for those 2 booleans, except `d` and `e` should be initialized with:
```
int d = input.nextInt();
int e = input.nextInt();
```
because `d` and `e` are int numbers and it won't compile using `nextBoolean()` (this method returns a boolean and we want a int)
Then, in the `if` statement you can **negate** the entire expression with negation operator `**!**` - this will apply the opposite logic. 
So:
```
if (!(e >= -10 && e <= 3)) {
	System.out.println("e is not in range"); // we change to a better information message, because now this is displayed.
}
```